### DISCUSSION TITLE
How vercel deploy this monorepo?

### DISCUSSION THREAD

**Discussion by codeacme17** [vercel/community#3798] [Help] [answered]
### Summary

I wanna figure out, how can vercel deploy my monorepo.

My project dir structure is:
```markdown
- app: vite + vue3
- server: python3.11 + fastAPI
- database: mysql.sql
```

and the repo address is [examor](https://github.com/codeacme17/examor)

deep holpfully who can give me a suggestion, i will very thank for! ❤

### Steps to Reproduce

none

**Comment by amyegan** [comment, marked as answer]
Hey @codeacme17. You can learn more about deploying Monorepos on Vercel in our docs: https://vercel.com/docs/monorepos

The database will need to be deployed to a database host, since it data writing won't work in the serverless environment. Or it could be converted to use [Vercel Postgres](https://vercel.com/docs/storage/vercel-postgres) if you prefer to avoid managing additional services.

Don't forget that you will need to [enable CORS](https://vercel.com/guides/how-to-enable-cors) or [use domain rewrites](https://vercel.com/guides/vercel-reverse-proxy-rewrites-external) to allow interactions between the app, server, and database.

**Reply by codeacme17** [reply]
Ok, thank u
